src/badge/README.md

```diff
@@ -2,17 +2,32 @@
 
 The `vscode-badge` component is used to highlight an item, attract attention, and/or flag status.
 
-## Attributes
-
-None
+![Badge hero](/docs/assets/images/badge-hero.png)
 
 ## Usage
 
-A `vscode-badge` can only contain numbers to follow the conventions of the Visual Studio Code design language.
+| ❌ Don't                                                                                           | ✅ Do                                                          |
+| -------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
+| ![Badge with text value](/docs/assets/images/badge-dont-1.png)                                     | ![Badge with number value](/docs/assets/images/badge-do-1.png) |
```

Comment:
If I'm pixel peeping all of these images are slightly different heights/widths from each other so you may want to consider switching all of these to HTML `img` tags with an explicitly defined `style` attribute

With that said, I think these are so close that I personally would just leave it